src/firestore/collection/collection.spec.ts

```diff
@@ -70,6 +69,19 @@ describe('AngularFirestoreCollection', () => {
 
     });
 
+    it('should optionally map the doc ID to the emitted data object', async (done: any) => {
+      const ITEMS = 1;
+      const { ref, stocks, names } = await collectionHarness(afs, ITEMS);
+      const idField = 'myCustomID';
+      const sub = stocks.valueChanges({idField}).subscribe(data => {
```

Comment:
Shouldn't `{idField}` be `{'idField': idField}`?

Reply:
I believe they are destructuring the valueChanges object and grabbing just the `idField` property.

Reply:
`{idField}` is shorthand for `{idField: idField}`
https://stackoverflow.com/questions/34414766/ Shorthand property names (ES2015)